.github/workflows/geoserver.yml

```diff
@@ -29,16 +29,16 @@ jobs:
     - name: "Setting up Java"
       uses: actions/setup-java@v2
       with:
-        java-version: '11'
+        java-version: '8'
```

Comment:
@jemacchi looks like this would be the only caveat for this pr? was it intentional? in any case it'd need to keep using java 11

Reply:
@groldan you can keep it with java 11

Reply:
@jemacchi it's your PR, please amend it and re-request review.

Reply:
@groldan it's done (rollback), but it is still failing during build since with my credentials I'm not able to push images to docker.io. Check details [here](https://github.com/georchestra/georchestra/actions/runs/3253456862/jobs/5340743033)
![image](https://user-images.githubusercontent.com/1171099/195956866-43e52a08-01a8-47ae-bd66-c730114a8c8b.png)